tool/tsh/tsh_test.go

```diff
@@ -810,21 +810,12 @@ func TestSSHAccessRequest(t *testing.T) {
 	}, setHomePath(tmpHomePath))
 	require.NoError(t, err)
 
-	// log out and back in with no access request
+	// drop the access request
```

Comment:
What do you think about having the old and the new test?
It seems they target different flows

Reply:
updated the test to leave the old logout flow